README.md

```diff
@@ -20,21 +20,21 @@ ensemble, and allowing for growth on an as-needed basis.
 ### 1. Clone this repository: ###
 
 ```
-git clone git@github.com:cmelab/hoomd-organics.git
-cd hoomd-organics
+git clone git@github.com:cmelab/JankFlow.git
+cd JankFlow
 ```
 
 ### 2. Set up and activate environment: ###
 #### a. Using HOOMD-blue from conda:
 ```
 conda env create -f environment-cpu.yml
-conda activate hoomd-organics
+conda activate jankflow
 python -m pip install .
 ```
 
 ## Basic Usage
 Please checkout the [tutorials](tutorials) for a detailed description of
-how to use HOOMD-Organics and what functionalities it provides.
+how to use JankFlow and what functionalities it provides.
 
 ## Documentation
 Documentation is available at [https://hoomd-organics.readthedocs.io](https://hoomd-organics.readthedocs.io)
```

Comment:
Looks like this line needs to be updated.

Reply:
Yeah, waiting for the docs to build and get the new html url.